1.2.0/index.html

```diff
@@ -85,31 +80,33 @@
       	}
       ],
       localBiblio: {
+        "OPENWAYBACK-CDXJ": {
```

Comment:
I wonder if this should even be referenced, as this wasn't actually ever implemented.
The CDXJ format here was derived from pywb implementation, first mentioned here:
https://github.com/webrecorder/pywb/wiki/CDX-Index-Format
https://pywb.readthedocs.io/en/latest/manual/indexing.html#cdxj-index

I think the openwayback proposal was an alternative that never got implemented..